ui/retribution_paladin/presets.ts

```diff
@@ -112,44 +113,39 @@ export const APLP2Ret = PresetUtils.makePresetAPLRotation('P2 Ret/Shockadin', AP
 export const APLP3Ret = PresetUtils.makePresetAPLRotation('P3 Ret/Shockadin', APLP3RetJson, {
 	customCondition: player => player.getLevel() === 50,
 });
-export const APLP4RetTwist = PresetUtils.makePresetAPLRotation('P4 Ret Twist', APLP4RetJson, {
-	customCondition: player => player.getLevel() === 60,
-});
-export const APLP4RetTwist6pT1 = PresetUtils.makePresetAPLRotation('P4 Ret Twist 6pT1', APLP4RetTwisting6PcT1Json, {
-	customCondition: player => player.getLevel() === 60,
-});
-export const APLP4RetExodin = PresetUtils.makePresetAPLRotation('P4 Ret Exodin', APLP4RetExodinJson, {
+export const APLP5Exodin6CF = PresetUtils.makePresetAPLRotation('P5 Exodin 6CF', APLP5ExodinJson, {
 	customCondition: player => player.getLevel() === 60,
 });
-export const APLP4RetExodin6pT1 = PresetUtils.makePresetAPLRotation('P4 Ret Exodin 6pT1', APLP4RetExodin6PcT1Json, {
+export const APLP5SealStacking6CF = PresetUtils.makePresetAPLRotation('P5 Seal Stacking 6CF', APLP5SealStackingJson, {
 	customCondition: player => player.getLevel() === 60,
 });
-export const APLPP5Twisting4DRSlow = PresetUtils.makePresetAPLRotation('P5 Twist 4DR Slow 3.5-3.6', APLPP5TwistingSlowJson, {
+export const APLShockadin = PresetUtils.makePresetAPLRotation('P5 Shockadin', APLShockadinJson, {
 	customCondition: player => player.getLevel() === 60,
 });
-export const APLPP5Twisting4DRSlower = PresetUtils.makePresetAPLRotation('P5 Twist 4DR Slower 3.7+', APLPP5TwistingSlowerJson, {
+export const APLP6Exodin = PresetUtils.makePresetAPLRotation('P6 Exodin', APLP6ExodinJson, {
 	customCondition: player => player.getLevel() === 60,
 });
-export const APLPP5Exodin = PresetUtils.makePresetAPLRotation('P5 Exodin', APLPP5ExodinJson, {
+export const APLP6OneHand = PresetUtils.makePresetAPLRotation('P6 1H Ret', APLP6OneHandJson, {
 	customCondition: player => player.getLevel() === 60,
 });
-export const APLPP5Shockadin = PresetUtils.makePresetAPLRotation('P5 Shockadin', APLPP5ShockadinJson, {
+export const APLTwisting = PresetUtils.makePresetAPLRotation('Twist', APLTwistingJson, {
 	customCondition: player => player.getLevel() === 60,
 });
 
 export const APLPresets = {
 	[Phase.Phase1]: [APLP1Ret],
 	[Phase.Phase2]: [APLP2Ret],
 	[Phase.Phase3]: [APLP3Ret],
-	[Phase.Phase4]: [APLP4RetTwist, APLP4RetTwist6pT1, APLP4RetExodin, APLP4RetExodin6pT1],
-	[Phase.Phase5]: [APLPP5Twisting4DRSlow, APLPP5Twisting4DRSlower, APLPP5Exodin, APLPP5Shockadin],
+	[Phase.Phase4]: [],
```

Comment:
Looks like the Phase 4 APLs are missing now?

Reply:
See thoughts from comment on gear sets.

Also, the twisting APLs would've been removed anyways and the P4 Exodin6pT1 is just seal stacking so they could've just been condensed into Phase5 APLs.

Reply:
Adding the relevant APLs back and trimming away those that can use later versions